code/obj/blob.dm

```diff
@@ -972,7 +979,7 @@ var/image/blob_icon_cache
 	dissipation_delay = 8
 	ks_ratio = 0.5
 	sname = "slime"
-	shot_sound = 'sound/impact_sounds/Slimy_Hit_3.ogg'
+	shot_sound = 'sound/voice/blob/blobshoot5.ogg'
```

Comment:
why 5

Reply:
5 sounds the best and I couldn't get a proc to work here

Reply:
the reason is because this is compile-time, if you don't want to spend the time to get it to work like your original intent was, delete the rest of the sounds and take off the 5